02.calendar/calendar.js

```diff
@@ -0,0 +1,26 @@
+import minimist from "minimist";
+const argv = minimist(process.argv.slice(2), {
+  alias: {
+    m: "month",
+    y: "year",
+  },
+});
+const now = new Date();
+const year = argv.year || now.getFullYear();
+const month = argv.month ? argv.month - 1 : now.getMonth();
+const firstDate = new Date(year, month, 1);
+const dayEnd = new Date(year, month + 1, 0);
+console.log(`${(month + 1).toString().padStart(8, " ")}月 ${year}`);
+console.log("日 月 火 水 木 金 土");
+process.stdout.write(" ".repeat(3 * firstDate.getDay()));
+for (let day = 1; day <= dayEnd.getDate(); day++) {
```

Comment:
```suggestion
console.log(`${(month + 1).toString().padStart(8, " ")}月 ${year}`);
console.log("日 月 火 水 木 金 土");
const firstDate = new Date(year, month, 1);
process.stdout.write(" ".repeat(3 * firstDate.getDay()));
const endDay = new Date(year, month + 1, 0).getDate();
for (let day = 1; day <= endDay; day++) {
```

変数の寿命を短くするのと、ループを回す前に月の最終日を取得しておく、という方法もありかなと思います。

[Doc 変数の寿命はできるだけ短くしよう \| FBC](https://bootcamp.fjord.jp/pages/311)